Training
Epoch 1/1 [Train]: 394it [02:14,  2.93it/s, Loss=0.0067]                                                                                           
Epoch 1/1 [Validation]:   0%|                                                                                              | 0/393 [00:00<?, ?it/s]
/home/vatsal/NWM/SSBware/senv/lib/python3.13/site-packages/numpy/_core/fromnumeric.py:3860: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/home/vatsal/NWM/SSBware/senv/lib/python3.13/site-packages/numpy/_core/_methods.py:145: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
Traceback (most recent call last):
  File "/home/vatsal/NWM/fno_sevir/fno_sevir.py", line 167, in <module>
    axes[1, i].set_title(f'Predicted Frame {i+1}')
      ^^^
NameError: name 'fig' is not defined
